pkg/cmd/attestation/verification/sigstore.go

```diff
@@ -150,14 +150,9 @@ func newCustomVerifier(trustedRootFilePath string) (*verify.SignedEntityVerifier
 	}
 
 	verifierConfig := []verify.VerifierOption{}
-	verifierConfig = append(verifierConfig, verify.WithSignedCertificateTimestamps(1))
 	verifierConfig = append(verifierConfig, verify.WithObserverTimestamps(1))
 
 	// Infer verification options from contents of trusted root
-	if len(trustedRoot.TimestampingAuthorities()) > 0 {
-		verifierConfig = append(verifierConfig, verify.WithSignedTimestamps(1))
-	}
-
```

Comment:
Conversely, this check prevents us from using a Sigstore public good compatible `--custom-trusted-root`.

Sigstore public good includes a timestamp authority section, which I believe is used by npm, but is definitely not used by artifact attestations in public repositories (which use Fulcio SCTs + Rekor inclusion proofs). Again, I believe it's safe to remove this check, as long as there is **some** `WithObserverTimestamps`, which we have above.

Reply:
Makes sense to me. Seems like a nice way to start to give users flexibility.